packages/extension-ui/src/components/Dropdown.tsx

```diff
@@ -57,7 +57,7 @@ function Dropdown({
   return (
     <div className={className}>
       <Label
-        $active
+        $small
```

Comment:
Does this have to be a transcient prop? If I see correctly, the underlying props are not passed-through to a DOM element, so I think this is not necessary?

Reasoning: apart of slight ick when starting a prop with `$`, imho differentiating transient props from regular props somewhat limits the solidity of a component by exposing internal mechanisms. Not a biggie of course, just a _polishing_ discussion.